engine/physics/src/physics/box2d_defold/box2d_defold_physics.cpp

```diff
@@ -559,9 +559,9 @@ namespace dmPhysics
                 {
                     b2Body* body_a = fixture_a->GetBody();
                     b2Body* body_b = fixture_b->GetBody();
-                    add_data.m_ObjectA = body_a;
+                    add_data.m_ObjectA = (uint64_t) body_a;
```

Comment:
Changed from a pointer to a handle, so that the new box2d handles can be stored.